jmx-metrics/src/main/resources/target-systems/kafka.groovy

```diff
@@ -14,81 +14,178 @@
  * limitations under the License.
  */
 
-def messagesInPerSec = otel.mbean("kafka.server:type=BrokerTopicMetrics,name=MessagesInPerSec")
-otel.instrument(messagesInPerSec, "kafka.messages.in", "number of messages in per second",
-        "1", "Count", otel.&longValueCallback)
+def messagesInPerSec = otel.mbean("kafka.server:type=BrokerTopicMetrics,name=MessagesInPerSec,topic=*")
+otel.instrument(messagesInPerSec,
+  "kafka.message.count",
+  "The number of messages received by the broker",
```

Comment:
Still per second though right?

Reply:
Surprisingly no, we've observed that the 'Count' attribute on the 'PerSec' MBeans are actually counters, so this is a total number of messages.